dtos/user_models.py

```diff
@@ -1,7 +1,9 @@
-from pydantic import BaseModel, validator, EmailStr
-from fastapi import HTTPException
+from datetime import datetime
+from pydantic import BaseModel, Field
 
 class user_signup_dto(BaseModel):
     name: str
     password: str
     email: str
+    created_at: datetime = Field(default_factory=datetime.now)
+    updated_at: datetime = Field(default_factory=datetime.now)
```

Comment:
Remove created_at and update_at from dto after given default value in user table